packages/twenty-server/src/modules/view/standard-objects/view-filter.workspace-entity.ts

```diff
@@ -50,6 +55,29 @@ export class ViewFilterWorkspaceEntity extends BaseWorkspaceEntity {
   })
   value: string;
 
+  @WorkspaceField({
+    standardId: VIEW_FILTER_STANDARD_FIELD_IDS.valueType,
+    type: FieldMetadataType.SELECT,
+    label: 'Value type',
+    description: 'View filter value type',
+    defaultValue: `'${ViewFilterValueType.STATIC}'`,
```

Comment:
Strange syntax no? Should it just be ` defaultValue: ViewFilterValueType.STATIC}`? Maybe you did this for a good reason but I don't see why?

Reply:
I think the quotes are needed to refer to an enum. Same syntax is used in `message-channel-message-association.workspace-entity.ts`. I'm not sure but I think without quotes the value would be interpreted as a string, I tried that and it didn't work. The generated table seems correct:

```
-- Generated by the database client.
CREATE TABLE workspace_1wgvd1injqtife6y4rvfbu3h5."viewFilter"(
    id uuid NOT NULL DEFAULT public.uuid_generate_v4(),
    "createdAt" timestamp with time zone NOT NULL DEFAULT now(),
    "updatedAt" timestamp with time zone NOT NULL DEFAULT now(),
    "deletedAt" timestamp with time zone,
    position double precision,
    name text NOT NULL DEFAULT 'Untitled'::text,
    "createdBySource" "viewFilter_createdBySource_enum" NOT NULL DEFAULT 'MANUAL'::"viewFilter_createdBySource_enum",
    "createdByWorkspaceMemberId" uuid,
    "createdByName" text NOT NULL DEFAULT ''::text,
    "displayValue" text NOT NULL DEFAULT ''::text,
    "fieldMetadataId" uuid NOT NULL,
    operand text NOT NULL DEFAULT 'Contains'::text,
    "value" text NOT NULL DEFAULT ''::text,
    "valueType" "viewFilter_valueType_enum" NOT NULL DEFAULT 'STATIC'::"viewFilter_valueType_enum",
    "viewId" uuid,
    PRIMARY KEY(id),
    CONSTRAINT FK_06858adf0fb54ec88fa602198ca FOREIGN key("viewId") REFERENCES view(id)
);
CREATE INDEX IDX_5653b106ee9a9e3d5c1c790419a ON workspace_1wgvd1injqtife6y4rvfbu3h5."viewFilter" USING btree ("deletedAt","viewId");
COMMENT ON TABLE workspace_1wgvd1injqtife6y4rvfbu3h5."viewFilter" IS '@graphql({"totalCount": {"enabled": true}})';
```

Reply:
OK great